pipelines/rj_cvl__osinfo_mongo/tasks.py

```
@@ -0,0 +1,1110 @@
+            file_id = futures[future]
+            try:
+                filename = future.result()
+                pdfs_created += 1
+
+                # Update processing_results with success
+                processing_results[file_id]["pdf_reconstructed_at"] = datetime.now()
+                processing_results[file_id]["pdf_reconstruction_failed"] = False
+
+                if pdfs_created % 50 == 0:
+                    print(f"  Reconstructed {pdfs_created}/{len(temp_files_map)} PDFs...")
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)